flixel/addons/editors/tiled/TiledTileSet.hx

```diff
@@ -92,23 +74,125 @@ class TiledTileSet
 			{
 				if (!node.has.id)
 				{
-					continue;
+					tileHeight = Std.parseInt(source.att.tileheight);
+				}
+				if (source.has.spacing) 
+				{
+					spacing = Std.parseInt(source.att.spacing);
+				}
+				if (source.has.margin) 
+				{
+					margin = Std.parseInt(source.att.margin);
 				}
 				
-				var id:Int = Std.parseInt(node.att.id);
-				tileProps[id] = new TiledPropertySet();
-				tileProps[id].keys.set("id", Std.string(id));
-				for (prop in node.nodes.properties)
+				// read properties
+				properties = new TiledPropertySet();
+				for (prop in source.nodes.properties) {
```

Comment:
Flixel's codestyle has a newline before the `{`. Although with a single line like this, omitting the brackets is also allowed.